src/main/java/tc/oc/pgm/prefix/PrefixRegistryImpl.java

```diff
@@ -28,19 +28,15 @@ public void onPrefixChange(PrefixChangeEvent event) {
     if (event.getUUID() == null) {
       return;
     }
-    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(event.getUUID());
-    if (matchPlayer == null
-        || matchPlayer.getBukkit() == null
-        || matchPlayer.getBukkit().isOnline()) {
+    final Player player = Bukkit.getPlayer(event.getUUID());
+    final MatchPlayer matchPlayer = PGM.get().getMatchManager().getPlayer(player);
```

Comment:
`final MatchPlayer player = PGM.get().getMatchManager().getPlayer(event.getUUID());` ?

Reply:
Does it matter if I use `Player#getBukkit`? I wrote it like this to avoid using that.

Reply:
What I'm saying is use what I put :)